### ISSUE TITLE
Error: Exception in callback of async function: TypeError: Cannot read property 'options' of undefined

### ISSUE THREAD

**Issue by adriamooney** [VeliovGroup/Meteor-iron-router-title#4]
I am getting this error, coming from `case !_.has(this.currentRoute.route.options, 'title'):`

**Comment by dr-dimitru** [comment]
Which error?
What are you passing?

**Comment by adriamooney** [comment]
It looks like it happens when I make an update to the code locally.  If I refresh the page then the error goes away.

**Comment by dr-dimitru** [comment]
Could you please post here a code which causes the error. And a error itself

**Comment by dr-dimitru** [comment]
is this issue still persists?

**Comment by adriamooney** [comment]
``` shell
Exception in callback of async function: TypeError: Cannot read property 'options' of undefined
    at IronRouterTitle.<anonymous> (http://localhost:3000/packages/ostrio_iron-router-title.js?865f455f55d84922fec5a47342195a7f950d7f2a:83:44)
    at IronRouterTitle.getTitle (http://localhost:3000/packages/ostrio_iron-router-title.js?865f455f55d84922fec5a47342195a7f950d7f2a:92:8)
    at null.<anonymous> (http://localhost:3000/packages/ostrio_iron-router-title.js?865f455f55d84922fec5a47342195a7f950d7f2a:67:19)
    at http://localhost:3000/packages/iron_router.js?7b16fe70bccff9182aca02afb2ccd7708ac57c64:1209:36
    at _.extend.withValue (http://localhost:3000/packages/meteor.js?43b7958c1598803e94014f27f5f622b0bddc0aaf:955:17)
    at hookWithOptions (http://localhost:3000/packages/iron_router.js?7b16fe70bccff9182aca02afb2ccd7708ac57c64:1208:27)
    at RouteController.runHooks (http://localhost:3000/packages/iron_router.js?7b16fe70bccff9182aca02afb2ccd7708ac57c64:271:7)
    at RouteController._runRoute (http://localhost:3000/packages/iron_router.js?7b16fe70bccff9182aca02afb2ccd7708ac57c64:557:8)
    at Function.Route.dispatch (http://localhost:3000/packages/iron_router.js?7b16fe70bccff9182aca02afb2ccd7708ac57c64:854:18)
    at route (http://localhost:3000/packages/iron_router.js?7b16fe70bccff9182aca02afb2ccd7708ac57c64:711:11)
```

**Comment by adriamooney** [comment]
error is:  `case !_.has(this.currentRoute.route.options, 'title'):` (line 83)  from this function: `IronRouterTitle.prototype.getTitle = function() {`

**Comment by dr-dimitru** [comment]
Your code sample, how do you use it within `Router`?

**Comment by adriamooney** [comment]
``` javascript
var companyName = Meteor.settings.public.ownerSettings.companyName;
    Router.configure({
        title: companyName
    });


   Router.route('/inventory', {
    name: 'inventory',
    title: companyName + ' | Inventory'
}); 
```

**Comment by dr-dimitru** [comment]
Meteor version, your platform (Mac/Linux/Win), IronRouter and this package version, please

**Comment by adriamooney** [comment]
iron router 1.0.9
ostrio:iron-router-title            1.0.0
mac

**Comment by dr-dimitru** [comment]
Meteor version

**Comment by adriamooney** [comment]
meteor version 1.1.0.3

**Comment by dr-dimitru** [comment]
hold on, I need somehow downgrade, it may take few days on my end

**Comment by dr-dimitru** [comment]
Hi @adriamooney 
Sorry for delayed response.
This issue was fixed in v1.0.1, see commit: https://github.com/VeliovGroup/Meteor-iron-router-title/commit/58d7825ef12d7269fc458754fdb67d6d0596c787#diff-6a7f5c833a5b6b47ae00f8e910fa5b29R23

Please update your dependencies
